pkg/resources/schema_grant_acceptance_test.go

```diff
@@ -38,6 +40,15 @@ func TestAccSchemaGrant(t *testing.T) {
 					resource.TestCheckResourceAttr("snowflake_schema_grant.test", "privilege", "USAGE"),
 				),
 			},
+			// TABLE AND VIEW FUTURE GRANTS
+			{
+				Config: futureTableAndViewGrantConfig(sName, roleNameTable, roleNameView),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_view_grant.select_on_future_views", "roles.#", "1"),
+					resource.TestCheckResourceAttr("snowflake_table_grant.select_on_future_tables", "roles.#", "1")
+					resource.TestCheckResourceAttr("snowflake_view_grant.select_on_future_views", "privilege", "SELECT"),
+			//	),
+			//},
```

Comment:
This does not appear to be syntactically correct.

Reply:
Yes, fixed in newer commit.